azure-pipelines.yml

```diff
@@ -36,5 +36,7 @@ steps:
     sudo apt-get install qemu qemu-system --fix-missing
   displayName: 'Download qemu'
   
-- script: zig*/zig build test -Drt-test=true -Dzig-path=zig*/zig
+- script: |
```

Comment:
Surely you would also have a normal build as well, as adding the testing and setting to true would have the OS be in testing mode when released. Then with the build OS with runtime testing enabled, add a command to start the kernel which would trigger the runtime tests

Reply:
There is actually a normal build without runtime testing enabled earlier on in the pipeline, check the "Build kernel" step.